### PR TITLE
Opmfilters

### PR DESCRIPTION
**Pull request by sravfeyn**
Complete fix for http://manage.dimagi.com/default.asp?177779#988465

### CODE REVIEW

**Comment by sravfeyn** [comment]
@czue @esoergel pretty urgent.

**Comment by millerdev** [comment]
:+1:

**Comment by czue** [comment]
:+1: from me. i'll let @esoergel merge since he has more context

File: custom/opm/utils.py [hunk 1]
```diff
@@ -30,7 +30,8 @@ def _safeint(value):
 
 class UserSqlData(SqlData):
     table_name = "fluff_OpmUserFluff"
-    group_by = ['doc_id', 'awc', 'awc_code', 'gp', 'block']
+    group_by = ['doc_id', 'name', 'awc', 'awc_code', 'bank_name',
+                'ifs_code', 'account_number', 'gp', 'block', 'village', 'gps']
```
Comment (esoergel) [inline custom/opm/utils.py]: lol.  THAT should be unique, at least.

File: custom/opm/reports.py [hunk 2]
```diff
@@ -700,7 +700,7 @@ def row_objects(self):
         """
         rows = []
         self._debug_data = []
-        awc_codes = {user['awc']: (user['awc_code'], user['gp'])
```
Comment (esoergel) [inline custom/opm/reports.py]: :+1:

**Pull request merged**